client/app/views/steps/subviews/progression.coffee

```diff
@@ -0,0 +1,4 @@
+{LayoutView} = require 'backbone.marionette'
+
+module.exports = class ProgressionView extends LayoutView
```

Comment:
`ItemView` is the regular way offered by Marionette for _simple_ views. So if we want to to it is a proper way, I recommend this one